ui/apps/platform/src/Containers/MainPage/InviteUsersModal.tsx

```diff
@@ -10,17 +19,26 @@ import { actions as inviteActions } from 'reducers/invite';
 import { actions as authActions } from 'reducers/auth';
 import { actions as groupActions } from 'reducers/groups';
 import { actions as roleActions } from 'reducers/roles';
+import { accessControlBasePath } from 'routePaths';
 import { AuthProvider } from 'services/AuthService';
+import { updateOrAddGroup } from 'services/GroupsService';
 import { dedupeDelimitedString } from 'utils/textUtils';
 import { mergeGroupsWithAuthProviders } from '../AccessControl/AuthProviders/authProviders.utils';
 import InviteUsersForm from './InviteUsersForm';
+// eslint-disable-next-line import/no-cycle
+import InviteUsersConfirmationNoEmail from './InviteUsersConfirmationNoEmail';
 
 type InviteFormValues = {
     emails: string;
     provider: string;
     role: string;
 };
 
+export type EmailBuckets = {
+    new: string[];
```

Comment:
Thank you, ECMAScript, for allowing keywords as property keys.

However, can you see if it is as gracious if you destructure `new` and then use it as a variable?